### ISSUE TITLE
2.14.2: libxml errors after patch release update

### ISSUE THREAD

**Issue by ro0NL** [doctrine/doctrine2#10626]
```
  - Upgrading doctrine/orm (2.14.1 => 2.14.2)
```

this PATCH release breaks our tree entities:

```
+ bin/console doctrine:migrations:migrate --allow-no-migration -v -n

In MappingException.php line 1005:
                                                                                                                                                                                              
  [Doctrine\ORM\Mapping\MappingException]                                                                                                                                                     
  libxml error: Element '{http://gediminasm.org/schemas/orm/doctrine-extensions-mapping}tree': No matching global element declaration available, but demanded by the strict wildcard.         
   in /app/config/doctrine/Entity.Some.orm.xml at line 18   
```

**Comment by greg0ire** [comment]
Duplicate of https://github.com/doctrine/DoctrineBundle/issues/1643

**Comment by ro0NL** [comment]
i guess. the breakage on our side was solely due `Upgrading doctrine/orm (2.14.1 => 2.14.2)`

**Comment by greg0ire** [comment]
Kinda… it should have happened when upgrading doctrine bundle to 2.9, but didn't because of a bug [fixed in ORM 2.14.2](https://github.com/doctrine/orm/pull/10579)
